tests/system_tests_one_router.py

```diff
@@ -1552,6 +1552,14 @@ def check_if_done(self):
             self.conn.close()
         elif self.n_sent == self.n_released:
             self.error = None
+
+            if not self.error:
```

Comment:
This 'if' is unnecessary.  Also, this test doesn't send pre-settled deliveries.  Why does it expect a count of dropped pre-settleds?

Reply:
Yes, the if statement is unnecessary. It should be removed.

It is true that the test does not send pre-settled deliveries. 

The router has allowUnsettledMulticast: yes
in this specific case we send an unsettled multicast to the router. But there is no receiver, so the router sends back the following to the client - 

[0x7ffb1c014c50]:0 -> @disposition(21) [role=true, first=0, last=0, settled=true, state=@released(38) []]

Hence this delivery can be considered a dropped pre-settled?

https://github.com/apache/qpid-dispatch/blob/master/src/router_core/transfer.c#L783